promote install instructions to top level menu item, enable mkdocs gh action

geoffcline opened the pull request.

geoffcline commented:
draft new page visible here

https://geoffcline.github.io/aws-load-balancer-controller/install/installation/

geoffcline commented:
ready for review @M00nF1sh :)

M00nF1sh commented on .github/workflows/mkdocs-deploy.yml:
```diff
@@ -0,0 +1,32 @@
+# GCLINE@ 23NOV2020
```
guess this should be amazon's copy right or leave empty :D

geoffcline replied:
I meant that more as a "who wrote this" but I can see the confusion, I'll fix

M00nF1sh commented on .github/workflows/mkdocs-deploy.yml:
```diff
@@ -0,0 +1,32 @@
+      - name: Checkout master
+        uses: actions/checkout@v2
+
+      - name: Deploy docs
```
we are using [mike](https://github.com/jimporter/mike) to deploy our multi-version docs.
I think we need to have our own actions or update `mhausenblas/mkdocs-deploy-gh-pages` to support mike.

geoffcline replied:
I'll take a look at how difficult that would be ... hopefully we can still do CD

M00nF1sh commented:
btw, how about the following layout
```
Welcome
Deployment
    Installation Guide
    Configurations
    Subnet Discovery
    Pod Readiness Gate
    Upgrade
        Migrate v1 to v2
User Guide
    Ingress
        ...
    Service
        ...
    TargetGroupBinding
        ...
    Tasks
        ...
Examples
    EchoServer
```

geoffcline commented:
ok, I can work using that layout tomorrow 👍

geoffcline commented:
Ok @M00nF1sh -  I refactored based on your suggested nav.

I also removed the github action, until I can get an action for mike working. 

Thanks!
Geoffrey

M00nF1sh commented:
/lgtm..would you help rebase this.

M00nF1sh commented:
/lgtm
